common/src/main/java/com/wynntils/models/character/CharacterSelectionModel.java

```diff
@@ -22,9 +34,16 @@ public final class CharacterSelectionModel extends Model {
     private static final StyledText CREATE_CHARACTER_NAME = StyledText.fromString("§a§lCreate a Character");
     private List<Integer> validCharacterSlots = new ArrayList<>();
     private List<ItemStack> selectionScreenItems = new ArrayList<>();
+    private ClassType currentCharacterClass = ClassType.NONE;
+    private boolean isReskinned = false;
+    private int currentCharacterLevel = 1;
+    private boolean isCreatingCharacter = false;
 
     public CharacterSelectionModel() {
         super(List.of());
+
+        Handlers.ActionBar.registerSegment(new CharacterSelectionClassSegmentMatcher());
+        Handlers.ActionBar.registerSegment(new CharacterSelectionLevelSegmentMatcher());
```

Comment:
Perhaps register `CharacterCreationSegmentMatcher` and `CharacterSelectionSegmentMatcher` here (and move them to this package). What do you think?